docs/news.rst

```diff
@@ -7,6 +7,15 @@
 ``psycopg`` release notes
 =========================
 
+Future releases
```

Comment:
Thank you for the care! :+1:

Reply:
I looked at the file history since I did not know what was the format when there are unreleased changes